features/src/rapids-build-utils/opt/rapids-build-utils/manifest.yaml

```
@@ -147,18 +147,23 @@ repos:
 
 - name: cuml
   path: cuml
-  git: {<<: *git_defaults, repo: cuml}
+  # TODO(jameslamb): revert this before merging
+  git: {<<: *git_defaults, repo: cuml, upstream: jameslamb, tag: libcuml-wheels}
```

```suggestion
  git: {<<: *git_defaults, repo: cuml}
```

Revert before merging.